keel/Core/constants.py

```diff
@@ -0,0 +1,3 @@
+
+PROJECT_PATH = os.path.abspath(os.path.join(os.path.dirname( __name__ ), '..'))+'/'
```

Comment:
is this the same as root dir .....already defined in settings
also why evaluate this every time....we can move such settings to config (base.py) , so that they get evaluated and available throughout the runtime!